project_proposals/RCM OSS.md

```diff
@@ -1,26 +1,125 @@
+Github - procedure needs to be implemented - dependencies outside Github (SCS) SCS: Software Development Company on beahlf of SBB
+##
+### What is the project's roadmap?
+up to 2024:
+* RCM-DX published on Github
+* RCM-DX Viewer available as freeware incl. sample data
+* Website www.sbb.ch/en/rcm-dx
+* Community: ~10 companies and several universities are using RCM-DX Viewer and format in Switzerland
+* Specification of RCM-DX format in tender in CH/EU for sourcing new diagnostic vehicles  <br>
 
-As a project intending to join the OpenRail Association, please fill out this questionnaire and send it to the Technical Committee. See the [incubation process](incubation_process.md) for details about criteria and requirements for new projects and how it works to get new projects into the association.
-
-## Questions
-
-* What is the project's name?
-* Describe the project. What does the project do, why is it valuable, where does it come from?
-* Who are the maintainers of the project (these will be the primary contacts for the OpenRail Association)?
-* Which organizations are sponsoring/contributing to the project?
-* Where is the code hosted?
-* What is the project's main license?
-* What other licenses does the project use, e.g. for included 3rd party code?
-* Are any trademarks associated with the project?
-* Does the project have a web site? Where is it?
-* What are the communication channels the project uses (such as mailing lists, Slack, IRC, etc.)?
-* What is the project's leadership team and decision-making process?
-* How is development of the project planned and organized? Is this transparent to the public?
-* What is the project's roadmap?
-* What other organizations in the world should be interested in this project?
-* Why would this project be a good candidate for inclusion in the OpenRail Association?
-* What is the project's plan for growing in maturity if accepted within the OpenRail Association?
+2024:
+* Publication of design for open source RCM-DX R/W library
+* Publication of RCM Viewer manual<br>
 
+2025:
+* RCM-DX open source RW library
+##
+### What other organizations in the world should be interested in this project?
+Several Railway Companies have expresses an interest but nothing concrete for now. We are in regular discussions with Infrabel about RCM-DX.
+##
+### Why would this project be a good candidate for inclusion in the OpenRail Association?
+The aim of RCM OSS is to make rail condition data easy to access, store and visualize. Therefore, RCM OSS is beneficial to railway companies as well as measurement system suppliers.
+##
+### What is the project's plan for growing in maturity if accepted within the OpenRail Association?
+Being hosted by OpenRail would makes RCM OSS a neutral project for the railway sector and not only an SBB project.
+##
+### Which of the repositories in the OSRD GitHub Org would you transfer to OpenRail? All of them or only a subset?
```

Comment:
### Which of the repositories in the RCM GitHub Org would you transfer to OpenRail? All of them or only a subset?